sound/soc/sof/intel/hda-dai.c

```diff
@@ -423,93 +423,79 @@ struct snd_soc_dai_driver skl_dai[] = {
 {
 	.name = "SSP0 Pin",
 	.playback = {
-		.stream_name = "ssp0 Tx",
 		.channels_min = 1,
```

Comment:
@bardliao The patch is good, but I'd like a bit more about the problem this is solving in commit message. I.e. why do we want to make sure the widgets are created by topology?